py3-jupyter-events.yaml

```diff
@@ -30,8 +30,8 @@ environment:
 pipeline:
   - uses: fetch
     with:
-      expected-sha256: 7be27f54b8388c03eefea123a4f79247c5b9381c49fb1cd48615ee191eb12615
-      uri: https://files.pythonhosted.org/packages/3f/0a/1c839290324ab93dc79950eaf26e198578db8b27edb587082b6061f4f9f5/jupyter_events-${{package.version}}.tar.gz
+      expected-sha256: fda08f0defce5e16930542ce60634ba48e010830d50073c3dfd235759cee77bf
+      uri: https://files.pythonhosted.org/packages/bb/1c/4b886f1312ed13d248acbee26066db759473b3c3a24244bdc211914761eb/jupyter_events-${{package.version}}.tar.gz
```

Comment:
Just last night I figured we can use a URI that doesn't contain a sha.  This means future auto update PRs should just work.  I.e. I think we can change this uri to be
```
uri: https://files.pythonhosted.org/packages/source/j/jupyter_events/jupyter_events-${{package.version}}.tar.gz
```

It seems the format to replace URIs that currently contain a sha is replace with

```
https://files.pythonhosted.org/packages/FIRST_CHAR_OF_PACKAGE_NAME/PACKAGE_NAME/PACKAGE_NAME-${{package.version}}.tar.gz
```
Wonder how easy it is to change melange convert to automatically do this rather than use the sha? ;) https://github.com/chainguard-dev/melange/blob/d0d501de96bcfae7d41cb69e29cd8711bb7b20c6/pkg/convert/python/python.go#L416

Reply:
let me test it

Reply:
@cpanato @rawlingsj this should cover most of the python stuff  https://github.com/wolfi-dev/os/pull/7368

Reply:
should I close this PR?